### DISCUSSION TITLE
pyg-lib conda package?

### DISCUSSION THREAD

**Discussion by cht74** [pyg-team/pyg-lib#273] [General]
Hello,

From

https://anaconda.org/pyg/repo

I can see that conda packages have been created (thank you!) for almost all additional pyg libraries - except for pyg-lib. Do you plan on making conda packages available for pyg-lib, too?

Thank you,
Christian

**Comment by rusty1s** [comment]
Yes, hopefully soon.

**Reply by cht74** [reply]
Great, thanks in advance.

**Comment by jakirkham** [comment]
There are packages for [`pyg-lib` on `conda-forge`]( https://anaconda.org/conda-forge/pyg-lib )

Here is the feedstock: https://github.com/conda-forge/pyg-lib-feedstock

Would welcome contributions from maintainers and the broader community there

**Reply by akihironitta** [reply]
We recently updated our repo at https://anaconda.org/pyg/pyg. Let us know if you encounter any issues!
